.github/workflows/ci.yml

```diff
@@ -109,21 +122,29 @@ jobs:
     needs: [ci]
     if: github.event_name != 'pull_request' && (startsWith(github.ref, 'refs/tags/v'))
     runs-on: ubuntu-22.04
-    env:
-      STTP_NATIVE: 1
+    strategy:
+      matrix: 
```

Comment:
hopefully this will work and the two publishes will publish separate jars - no other way to find out than try ;)

Reply:
Yeah, unfortunately there's no nice way to do a dry-run. We could probably make `publishArtifacts` dependent on some variables, but I'm not sure if it's worth adding complexity.